OG_Format.adoc

```diff
@@ -268,8 +268,8 @@ OG1.0 requirements cover positioning variables and geolocation of any scientific
 * data type: double
 * dimension: N_MEASUREMENTS |
 
-* long_name = “longitude of each measurement and GPS location”;
-* standard_name = “longitude”;
+* long_name = “Longitude east”;
+* standard_name = “Longitude”;
```

Comment:
I believe the standard name `longitude` is defined with all lower case. Look at: http://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html